.flake8

```diff
@@ -39,7 +39,8 @@ exclude =
     __pycache__,
     tests/*,
     venv,
-    manage.py
+    manage.py,
+    demoproj/*
```

Comment:
Excluded demo project from flake8, but can remove this before finalizing if we want.